src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -233,10 +233,12 @@ const MetaTraderConfig = (() => {
             mt5_new_account : 1,
             name            : 'test real labuan advanced',
         };
-        BinarySocket.send(req);
-
-        // update account status authentication info
-        BinarySocket.send({ get_account_status: 1 }, { forced: true });
+        BinarySocket.send(req).then((response) => {
+            if (!response.error) {
```

Comment:
but the point is that after it gives us error, we should update account status, right?

Reply:
omg yes youre right 😅